lib/travis/build/git/clone.rb

```diff
@@ -72,6 +75,18 @@ def config
             data.config
           end
 
+          def write_netrc
+            sh.newline
+            sh.echo "Using $HOME/.netrc to clone repository.", ansi: :yellow
+            sh.newline
+            sh.raw "echo -e \"machine github.com\n  login #{data.token}\\n\" > $HOME/.netrc"
```

Comment:
Does the `machine github.com` need to be customized for the GitHub install? For instance should this read 

```
machine ghe.volcanicislandfortress.com
```

for my GitHub Enterprise install?